### PR TITLE
IRGN - add traffic sign prefix

### PR DESCRIPTION
**Pull request by andreo141**
### Overview
Governments want to be able to see the type of traffic sign prefixed in the generated rdfa when adding a traffic measure.

##### connected issues and PRs:
https://binnenland.atlassian.net/browse/GN-4882

### How to test/reproduce
Try to add a traffic sign. Depending on the type of sign the prefix should change.

### Challenges/uncertainties
Not super familiar with IRGN and the editor, so all feedback is welcome. Based on the ticket description, this is what i understood that needed to happen.

### Checks PR readiness
- [x] UI: works on smaller screen sizes
- [x] UI: feedback for any loading/error states
- [x] Check if dummy app is correctly updated
- [x] Check cancel/go-back flows
- [x] changelog
- [x] npm lint
- [x] no new deprecations

### CODE REVIEW

**Review by lagartoverde — CHANGES_REQUESTED** [review]

**Comment by andreo141** [comment]
I modified the query where the signs are being fetched and included the signtype in the SELECT portion of the query.

File: addon/models/sign.ts
Comment (lagartoverde) [inline addon/models/sign.ts:7]: You can rename this type as it's already inside the sign :)

**Review by lagartoverde — APPROVED** [review]
Works for me, weird that we don't add this type as rdfa info but indeed the ticket doesn't ask for that, maybe @nvdk can throw some light?

File: addon/models/sign.ts
Comment (nvdk) [reply addon/models/sign.ts]: that's ok for now

**Comment by andreo141** [comment]
This can be merged but it seems like pipeline is still waiting for approval

**Pull request merged**
